### ISSUE TITLE
Highlight occurrences of the same word in decompilation when selected

### ISSUE THREAD

**Issue by Col-E** [Col-E/Recaf#811]
This would facilitate clicking on a field or method name showing where all the other uses of that name are.

**Comment by xroberx** [comment]
Yes, please, this is a must when working on large obfuscated files.

**Comment by V4LK0y** [comment]
I can take on this if it's still available, really need it

**Comment by Col-E** [comment]
Added a while ago and forgot to close: 27ce03c8b74a3cafa488d25381bbd73a3ed08754

<img width="1321" height="478" alt="Image" src="https://github.com/user-attachments/assets/05ade319-80bd-45f7-99cf-9cedf6500cde" />
